src/components/deploymentConfig/types.tsx

```diff
@@ -114,6 +115,11 @@ export interface DeploymentConfigFormCTAProps {
     reload: () => void
     isValues?: boolean
     convertVariables?: boolean
+    openLockedDiffDrawer: () => void
+    isSuperAdmin: boolean,
+    showLockedDiffForApproval: boolean
+    setShowLockedDiffForApproval: (show: boolean) => void
+    checkForLockedChanges: (saveEligibleChanges:boolean) => any
```

Comment:
check if the return type can be more specific

Reply:
done